openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -1,3 +1,10 @@
+"""
+Contains methods for accessing weekly course highlights. Weekly highlights is a
+ScheduleExperience built on the Schedules app. The Schedules app introduces
+dynamic pacing in self-paced courses.
```

Comment:
You can remove this last sentence - as it's redundant with the information already available in the Schedules README file.